pom.client.xml

```diff
@@ -679,6 +701,31 @@
               </compilerArgs>
             </configuration>
           </plugin>
+          <!-- Surefire plugin is broken, https://issues.apache.org/jira/browse/SUREFIRE-1501 -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${maven-surefire-plugin.version}</version>
+            <configuration>
+              <argLine>
+                <!-- KeyVault tests fail without these exports -->
+                --add-exports com.azure.core/com.azure.core.implementation.http=ALL-UNNAMED
```

Comment:
Are these issues that we need to fix later? If so, maybe creating issues for them help.

Reply:
This is a section I would like to dive more deeply into, to make sure we need all of these (and can't, e.g. export them in the module-info)

Reply:
Yes, I will report an issue for this.

Reply:
I am fine with that - let's get two issues, one for module-info `exports .. to` statements, and the other for the build-time `add-opens` statements. Once those are filed I am fine with merging what we have today.